lib/Doctrine/DBAL/TransactionMode.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
+
+final class TransactionMode
```

Comment:
Since this class defines transaction isolation levels, would it make sense to name it accordingly?

Reply:
Makes sense. So `TransactionIsolation` or `TransactionIsolationLevel`?

Reply:
FWIW, I'd personally prefer `TransactionIsolationLevel`.